.changeset/dirty-keys-join.md

```diff
@@ -0,0 +1,7 @@
+---
+"@vocabulary/earl": major
+"@zazuko/prefixes": major
+"@zazuko/vocabularies": major
```

Comment:
I marked this change as breaking but I think this could as well be "patch" because the wrong namespace was a mistake after all and should be fixed in any current consumer